Add Beatmap Spotlights winter 2023 news post & wiki article (#8427)
ppy/osu-wiki: news/2022-12-04-beatmap-spotlights-season-8-winter-2023-precision.md

*described by [SadEgg](https://osu.ppy.sh/users/10278243)*

This spectacle of a map has been around the osu!catch community for nearly a decade now and it does not disappoint. CLSW took a beautiful instrumental masterpiece - and as the mapping paragon they are - built one themselves.

This 2013 200bpm showstopper challenges your mastery of the platter in many different aspects including tap dashes, anti-flow streams, and of course some of those signature wiggles that even test top players’ finger-control. Exhibiting all of those patterns cherry-picked and slotted in such a way showcasing their variation to dense and technical mapping through unique streams and flow. Synergy with each emphasis, this map fits the song perfectly making for a display and an exceedingly polished map.

### osu!mania

#### Players' Pick

![](/wiki/shared/news/2022-12-04-beatmap-spotlights-season-8-winter-2023-precision/3587748.jpg)

**[laura les - Haunted (dive to the heart remix) [[4K] Another]](https://osu.ppy.sh/beatmapsets/1683972#mania/3587748)**, mapped by [Monheim](https://osu.ppy.sh/users/6814203)\
*primarily suggested and described by [Kibitz](https://osu.ppy.sh/users/7418493)*

> The Chord Jack Ranked Maps Era is here! Monheim absolutely blasts the gates open with their FULL rice based mapset.
> Yes. You heard me right. Zero long notes. RICE PLAYERS REJOICE as this 154 bpm dense chord jack mapset challenges your stamina, consistency
> and raw reading skill.

Reviewer: ```suggestion
The Chord Jack Ranked Maps Era is here! Monheim absolutely blasts the gates open with their FULL rice-based mapset. Yes. You heard me right. Zero long notes. RICE PLAYERS REJOICE as this 154 BPM dense chord jack mapset challenges your stamina, consistency and raw reading skill.
```

> The *Another* difficulty in particular features a massive step up from the lower difficulties but is still manageable for intermediate
> players looking for a tough challenge! Monheim executes this difficulty swimmingly with an abundance of quads matching the everlasting bass coupled with stacked notes and minijacks and a bunch of 1/4 quadstreams to give this map that extra kick. This map has a very unique concept which is rare to see be done well, as the balance between both the chord-jacks and vocal dump feels really well polished. This is definitely a leading factor in a potential shift we may see in the future of osu! mania's ranked section.

Reviewer: ```suggestion
The Another difficulty in particular features a massive step up from the lower difficulties but is still manageable for intermediate players looking for a tough challenge! Monheim executes this difficulty swimmingly with an abundance of quads matching the everlasting bass coupled with stacked notes and minijacks and a bunch of 1/4 quadstreams to give this map that extra kick. This map has a very unique concept which is rare to see be done well, as the balance between both the chord jacks and vocal dump feels really well polished. This is definitely a leading factor in a potential shift we may see in the future of osu!mania's Ranked section.
```

#### Curators' Pick

![](/wiki/shared/news/2022-12-04-beatmap-spotlights-season-8-winter-2023-precision/3492128.jpg)

**[Camellia - Newspapers for Magicians [Extra]](https://osu.ppy.sh/beatmapsets/1645306#mania/3492128)**, mapped by [Blocko](https://osu.ppy.sh/users/4075092)\
*primarily suggested and described by [Antalf](https://osu.ppy.sh/users/8793773)*

A timeless artist for rhythm games, Camellia delivers another banger song with an amazing mapset by one of our veteran community members, Blocko! This map includes an amazing spread that caters to every player no matter the skill level and will guarantee a memorable experience regardless.

> You will find yourself enveloped from easy to follow rhythms up to insane and complex long notes while playing this map. They both intertwine in harmony to create the amazing *Magic* last difficulty, but don’t sleep on the lower ones! They are as great as it gets with subtle long note and a variety of other patterns that’ll make you want not just to keep playing but vibe along with the music.

Reviewer: ```suggestion
You will find yourself enveloped from easy-to-follow rhythms up to insane and complex long notes while playing this map. They both intertwine in harmony to create the amazing last difficulty, Magic, but don't sleep on the lower ones! They are as great as it gets with subtle long note patterns and a variety of other ones that'll make you want not just to keep playing but vibe along with the music.
```

Blocko, even after years of mapping and contribution, is still consistently delivering amazing charts and sets as a whole, making unique maps along with an experience that is due to last.

### Full Players' Pick voting results

> Check out each respective game mode's player voting full results:

Reviewer: ```suggestion
Check out each respective game mode's player voting results:
```

- [osu!](https://osu.ppy.sh/community/contests/156)
- [osu!taiko](https://osu.ppy.sh/community/contests/157)
- [osu!catch](https://osu.ppy.sh/community/contests/158)
- [osu!mania](https://osu.ppy.sh/community/contests/159)

## Inside the Spotlights Team

This season, we will say farewell to some of our members as well as welcome some fresh blood to the ranks of Spotlights Curators. Thanks for all your hard efforts to deliver the seasonal dose of outstanding beatmap material!

**Curator additions:**

- **osu!taiko**: [Horiiizon](https://osu.ppy.sh/users/8071438)
- **osu!mania**: [MadBricktree](https://osu.ppy.sh/users/4790958), [Maxus](https://osu.ppy.sh/users/4335785)

> [Nowaie](https://osu.ppy.sh/users/5428909) will also be stepping down from leading the osu! Curators team, [ChillierPear](https://osu.ppy.sh/users/9501251) will handle his tasks from now on.

Reviewer: ```suggestion
[Nowaie](https://osu.ppy.sh/users/5428909) will also be stepping down from leading the osu! Curators team, while [ChillierPear](https://osu.ppy.sh/users/9501251) will handle his tasks from now on.
```

> For the full Spotlights Team members list, check out the [Beatmap Spotlights wiki article](/wiki/Beatmap_Spotlights).

Reviewer: ```suggestion
For the full Spotlights Team members list, check out the [Beatmap Spotlight Curators wiki article](/wiki/People/The_Team/Beatmap_Spotlights).
```

## Season schedule and leaderboards

| Event | Time |
| :-- | :-- |
| Season start | Dec 4, 2022 |
| Playlist lobby: Playlist A | Dec 4 - Dec 18, 2022 |
| Playlist lobby: Playlist B | Dec 18 - Jan 1, 2023 |
| Playlist lobby: Playlist C | Jan 1 - Sep 13, 2023 |
| Playlist lobby: Playlist A | Jan 13 - Jan 20, 2023 |
| Playlist lobby: Playlist B | Jan 20 - Jan 27, 2023 |
| Playlist lobby: Playlist C | Jan 27 - Feb 3, 2023 |
| Season conclusion | Feb 3, 2023 |

> The Winter Season 2023 leaderboard will be available over at the [Beatmap Spotlights Winter 2023](/wiki/Beatmap_Spotlights/Seasons/2023_Winter) wiki article once the first batch of lobbies conclude in the coming weeks.

Reviewer: ```suggestion
The Winter Season 2023 leaderboard will be available over at the [Beatmap Spotlights Winter 2023 wiki article](/wiki/Beatmap_Spotlights/Seasons/2023_Winter) once the first batch of lobbies conclude in the coming weeks.
```

| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/platinum_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/platinum_2.png) | Platinum | 3% - 10% |
| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/gold_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/gold_2.png) | Gold | 10% - 25% |
| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/silver_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/silver_2.png) | Silver | 25% - 50% |
| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/bronze_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/bronze_2.png) | Bronze | 50% - 70% |
| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/copper_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/copper_2.png) | Copper | 70% - 95% |
| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/iron_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2023/osu/iron_2.png) | Iron | 95% - 100% |

The percentage values may vary a little depending on the game mode and/or participant counts.

> The Rhythm Incarnate tier threshold is picked manually based on season's participant count and general size of other tiers, though it is an absolute number that ranges between top 2 and top 50 in most cases.

Reviewer: ```suggestion
The Rhythm Incarnate tier threshold is picked manually based on the season's participant count and general size of other tiers, though it is an absolute number that ranges between top 2 and top 50 in most cases.
```

- [Camellia - Fastest Crash (\_Hikari) \[\[4K\] Hard\]](https://osu.ppy.sh/beatmapsets/1725902#mania/3535122), suggested by [Crumpey](https://osu.ppy.sh/users/3518705)
- [linear ring - Can you hear me? (Ulis-) \[\[4K\] Yeah (Insane)\]](https://osu.ppy.sh/beatmapsets/1621762#mania/3311142), suggested by [Antalf](https://osu.ppy.sh/users/8793773)
- [Dictate - Empire (arccat) \[\[4K\] Civilization\]](https://osu.ppy.sh/beatmapsets/1718862#mania/3512696), suggested by [Protastic101](https://osu.ppy.sh/users/6712747)
- [mu\_akaru - Choukousoku Henbyoushi no Alien (MJH) \[\[5K\] Timeshift\]](https://osu.ppy.sh/beatmapsets/1324800#mania/2744040), suggested by [MadBricktree](https://osu.ppy.sh/users/4790958)
- [goreshit - o'er the flood (Jinjin) \[\[7K\] jinjin's extra\]](https://osu.ppy.sh/beatmapsets/454793#mania/975161), suggested by [MadBricktree](https://osu.ppy.sh/users/4790958)
- [paraoka - L9 (Jesuren) \[4K Daydream\]](https://osu.ppy.sh/beatmapsets/1565681#mania/3197091), suggested by [Hydria](https://osu.ppy.sh/users/808176)

> *osu!mania playlists will include 7 beatmaps instead of 9 this season.

Reviewer: ```suggestion
*osu!mania playlists will include 7 beatmaps instead of 9 this season.*
```

Playlist B and C will be available on the [Beatmap Spotlights article](/wiki/Beatmap_Spotlights/Seasons/2022_Winter) as we open the playlist lobbies, so stay tuned!

> ## The Future

Reviewer: ```suggestion
## The future
```

> The new Beatmap Spotlights is still a project in its early development stage and depends a lot on community feedback and engagement to improve. Many things can (and will) change for future seasons, and it all depends on *your* feedback!

Reviewer: how much longer will it be in its "early development stage"? 🤔

> What do you want for the next season? What needs to be fixed? What are your opinions on the current system? You can voice out all your opinions in the [osu! community Discord](https://discord.gg/0Vxo9AsejDkGlk3H) (#beatmap-spotlights) and we will keep note of every suggestion, so please let us know how you feel about things.

Reviewer: ```suggestion
What do you want for the next season? What needs to be fixed? What are your opinions on the current system? You can voice out all your opinions in the [osu! community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H) (`#beatmap-spotlights` channel) and we will keep note of every suggestion, so please let us know how you feel about things.
```
